zetteldeft.el

```diff
@@ -221,10 +228,11 @@ and the file name (without extension) is added to the kill ring.
 When `evil' is loaded, enter insert state."
   (interactive (list (read-string "name: ")))
   (let* ((zdId (zetteldeft-generate-id))
-         (zdName (concat zdId " " str)))
+         (zdName (concat zdId zetteldeft-id-filename-separator str)))
+  (deft-filter str t)
```

Comment:
What's the reason for adding this line?

Reply:
I must have been looking at deft-new-file . I've removed that line locally and it's working fine for me - will update.